src/test/java/calculator/StringAddCalculatorTest.java

```diff
@@ -35,13 +35,13 @@ public class StringAddCalculatorTest {
 
     @ParameterizedTest
     @ValueSource(strings = {"-1,2,5", "1,-5,6", "1:3,-8", "-1:-2:-8"})
-    void 음수를_전달할_경우_RuntimeException_발생(String input) {
-        assertThrows(RuntimeException.class, () -> StringAddCalculator.splitAndSum(input));
+    void 음수를_전달할_경우_IllegalStateException_발생(String input) {
```

Comment:
```suggestion
    void 음수를_전달할_경우_예외를_던진다(String input) {
```

굳이 특정 예외를 지칭하기보단 좀 더 포괄적인 단어로 메서드와 로직간의 결합도를 떨어트릴 수 있어요.